tests/e2e/configuration/configuration_check.py

```diff
@@ -0,0 +1,33 @@
+# Copyright 2022 Memgraph Ltd.
+#
+# Use of this software is governed by the Business Source License
+# included in the file licenses/BSL.txt; by using this file, you agree to be bound by the terms of the Business Source
+# License, and you may not use this file except in compliance with the Business Source License.
+#
+# As of the Change Date specified in that file, in accordance with
+# the Business Source License, use of this software will be governed
+# by the Apache License, Version 2.0, included in the file
+# licenses/APL.txt.
+
+import typing
+import mgclient
+import sys
+import pytest
+
+import default_config
+
+
+def test_does_default_config_match():
+    connection = mgclient.connect(host="localhost", port=7687)
+    connection.autocommit = True
+
+    cursor = connection.cursor()
+    cursor.execute("SHOW CONFIG")
+    config = cursor.fetchall()
+
+    for flag in config:
+        assert config[1] == default_config.startup_config[1]
```

Comment:
This check is not good, it only checks the first config option. Maybe 1.) check their sizes are equal; 2.) iterator over them by index.

Reply:
Good catch, I don't know what I was thinking :smile:
Also added the size check.